src/main/java/com/jossemargt/cookietwist/exception/InvalidFormatException.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * The MIT License
+ *
+ * Copyright (C) 2018 Jonnatan Jossemar Cordero
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.jossemargt.cookietwist.exception;
+
+/**
+ * The Class InvalidFormatException is meant to describe any CookieValue Model
+ * de-serialization fault, mostly related to String formatting, numeric parsing
+ * or un-expected amount of String tokens.
+ */
+@SuppressWarnings("serial")
+public class InvalidFormatException extends Exception {
```

Comment:
Being honest, I was completely unaware of the difference between checked and unchecked Exceptions until you suggested this, and sure thing!  I'm going to change it and update the class/interface names into a more (not so fancy) Java way as well, based on the off PR conversation that we had during this week.

Reply:
Done.

BTW I'm trying to figure out if I should rename the `value` package to `transform` instead, since it contains the serialization objects, but still that new name doesn't sound alright for me. Do you have any suggestion?